src/collector.ts

```diff
@@ -105,13 +105,38 @@ class NaiveGomodParser {
     dependencies: Array<IDependency>;
 
     static parseDependencies(contents:string, goImports: Set<string>): Array<IDependency> {
+        let replaceDependencies = new Map();
+
+        function addReplaceDependency(line: string, index: number) {
```

Comment:
This function need to be moved out of the parent function.

Reply:
As a class method?

Reply:
as a static method.

Reply:
ACK!